certbot/client.py

```diff
@@ -52,9 +53,12 @@ def determine_user_agent(config):
     """
 
     if config.user_agent is None:
-        ua = "CertbotACMEClient/{0} ({1}) Authenticator/{2} Installer/{3}"
+        auto = "no" if cli.cli_command == "certbot" else "yes"
+        ua = ("CertbotACMEClient/{0} ({1}) Authenticator/{2} Installer/{3} Verb/{4} "
```

Comment:
As a comment on the authenticator, we should consider capturing the type (currently always `dns`, but that may change in the future).